module-system/sov-modules-stf-template/src/lib.rs

```diff
@@ -137,7 +138,8 @@ where
             .compute_state_update(log, &witness)
             .expect("Storage update must succeed");
 
-        self.current_storage.commit(&node_batch);
+        self.current_storage
+            .commit(&node_batch, &Default::default());
```

Comment:
Here, we also need to retrieve `AccessoryUpdate` from the working set and pass it to the commit, right?